### PR TITLE
fix: repo entities fields are now set when cascade save

### PR DESCRIPTION
**Pull request by osher-sade**

### CODE REVIEW

File: packages/polaris-test/nest-server-schema-first/graphql.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (chenshoo) [inline packages/polaris-test/nest-server-schema-first/graphql.ts]: dont commit this file

File: packages/polaris-typeorm/src/typeorm-bypasses/polaris-entity-manager.ts [hunk 2]
```diff
@@ -220,6 +204,42 @@ export class PolarisEntityManager extends EntityManager {
...
+            maybeEntityOrOptions.realityId = context?.requestHeaders?.realityId ?? 0;
+            PolarisEntityManager.setUpnOfEntity(maybeEntityOrOptions, context);
+        }
+
+        if (entityMetadata && entityMetadata.relations) {
+            for (const relation of entityMetadata.relations) {
+                const relationMetadata = relation.inverseEntityMetadata;
+                const isCascadeInsert = relation.isCascadeInsert;
+                const isCommonModel =
+                    relationMetadata.inheritanceTree.find(
```
Comment (chenshoo) [inline packages/polaris-typeorm/src/typeorm-bypasses/polaris-entity-manager.ts]: use isDescendentOfCommonModel
method

File: packages/polaris-typeorm/src/typeorm-bypasses/polaris-entity-manager.ts [hunk 3]
```diff
@@ -220,6 +204,42 @@ export class PolarisEntityManager extends EntityManager {
...
+                        (ancestor) => ancestor.name === 'CommonModel',
+                    ) !== undefined;
+                if (isCommonModel && isCascadeInsert) {
```
Comment (chenshoo) [inline packages/polaris-typeorm/src/typeorm-bypasses/polaris-entity-manager.ts:232]: check if isCascadeUpdate too

Comment (chenshoo) [reply packages/polaris-typeorm/src/typeorm-bypasses/polaris-entity-manager.ts:232]: call this from update

**Review by ronkatz96 — APPROVED** [review]

**Pull request merged**